src/hooks/use-play-card-audio.ts

```diff
@@ -42,6 +47,7 @@ export function usePlayCardAudio() {
   }
 
   function clearAudio() {
+    abortSpeechCapture();
     if (!activeAudioRef.current) {
       clearTimer();
```

Comment:
also a bug(?) i observed, if you pause during a wait period and go to a previous card, it stays flipped on the 'back'.

during the `wait2` period: `playing front|wait1|playing back|wait2`, i think going previous should flip the card to the front, could add something like this before we return here:
```ts
clearTimer();
if (activeCard && activeCardSide === 'back') {
    setActiveCardSide('front');
}
return;
```

Reply:
hmm, yeah there seem to be some bugs with playback. I added the snippet but was unable to test it because of the updated audio. I'll test it some more once we get it all merged.